src/crawl.js

```diff
@@ -0,0 +1,107 @@
+/*
+eslint no-await-in-loop: "off",
```

Comment:
this is on for a reason. you can keep it disabled if you really like, but it will likely hurt performance because you cant benefit from parallelization